boogiestats/templates/boogie_ui/song.html

```diff
@@ -14,6 +14,12 @@ <h2 class="mt-2">{% include "boogie_ui/song_display_name.html" %}</h2>
        class="btn {% if request.resolver_match.url_name == "song_highscores" %}btn-success{% else %}btn-secondary{% endif %}">
         Highscores ({{ num_highscores }})
     </a>
+    {% if user.is_authenticated %}
+        <a href="{% url "song_by_player" song_hash=song.hash player_id=user.player.id %}"
+           class="btn {% if request.resolver_match.url_name == "song_by_player" %}btn-success{% else %}btn-secondary{% endif %}">
+            My scores ({{ my_scores }})
+        </a>
+    {% endif %}
     <h3>
         {% if request.resolver_match.url_name == "song" %}
```

Comment:
The page still says `Highscores` when browsing `My scores` subpage

![image](https://user-images.githubusercontent.com/9034451/209433086-f7470b29-31ca-4628-8c74-775bb344530b.png)

Reply:
fixed, and at the same time I added different titles depending on who is looking at scores
if I'm looking at my scores then title says "My scores", if I'm looking at other people's scores title says "PlayerX scores"

Reply:
good idea!

Reply:
however, there's one issue with the button now:

![image](https://user-images.githubusercontent.com/9034451/209442626-01a59a5c-b32a-489d-ae96-08d2199fa948.png)

it indicates that I'm browsing *my* scores when in fact I'm looking at someone else

Reply:
You're right, it could end being confusing. I changed it so button will turn green only for my scores, and will stay gray for anyone else's scores.